pinot-common/src/main/java/com/linkedin/pinot/common/utils/time/DateTimeFieldSpecUtils.java

```diff
@@ -28,17 +28,17 @@
 
 public class DateTimeFieldSpecUtils {
```

Comment:
It would be nice to have a class TimeSpecFormat that takes in the format and provides all these getters.